docs/linux.md

```diff
@@ -44,7 +45,18 @@ Curl version 8.0.1
 	```
 	sudo yum remove libcurl-devel openssl-devel
 	```
+Following steps will replace the existing versions of OpenSSL and Curl from the system. If you want to keep the existing versions then use [Installation-Script](../utils/install_openssl_curl.sh) script to install Openssl and Curl at a different location.
```

Comment:
Can u add this note before "#### Steps to remove the older OpenSSL and curl packages" ?